corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -126,6 +145,112 @@
 
       </div>
 
+      {% if not creates_new_location %}
+        {% initial_page_data 'location' location %}
+
+        <div class="tab-pane {% if form_tab == "descendants" %}active{% endif %}" id="descendants-info">
+          <div class="panel panel-modern-gray panel-form-only">
+            <div class="panel-body">
+
+              <div class="row">
+                <div class="col-lg-4" style="clear: both">
+                  <div id="location_descendants_tree">
+                    <div data-bind="template: { name: 'location-template', if: pseudoRoot, data: pseudoRoot }"></div>
+                  </div>
+                </div>
+              </div>
+
+              <script type="text/html" id="location-template">
```

Comment:
is this copy/pasted from `locations.html`? Is there a way you can reuse the template between those pages? I believe you can make a `templates` html file and include it in both of these pages: 
e.g. 
https://github.com/dimagi/commcare-hq/blob/d7a4a7f17ba8d44db48114687b39afb17160d889/corehq/apps/cloudcare/templates/formplayer-common/base.html#L80

Reply:
That is a good idea